sdk/schemaregistry/Azure.Data.SchemaRegistry/tests/Samples/SchemaRegistryClientSamples.ReadmeSnippets.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.Data.SchemaRegistry.Tests.Samples
+{
+#pragma warning disable SA1649 // File name should match first type name
+    public class SchemaRegistryReadmeSnippets : SamplesBase<SchemaRegistryClientTestEnvironment>
+#pragma warning restore SA1649 // File name should match first type name
+    {
+        [Test]
+        public void CreateSchemaRegistryClient()
+        {
+            var endpoint = TestEnvironment.SchemaRegistryUri;
+            var credentials = TestEnvironment.Credential;
+
+            #region Snippet:CreateSchemaRegistryClient
+            //@@ string endpoint = "<event_hubs_namespace_hostname>";
```

Comment:
nit: this kills half of the purpose of the snipped being code

Reply:
Since I cannot run live tests yet (we cannot create the resources necessary), I've updated the snippets to simply compile the code and not use the `//@@` syntax.